mne/io/brainvision/brainvision.py

```diff
@@ -595,7 +595,7 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
             else:
                 info['highpass'] = float(highpass[0])
                 if hp_s:
-                    info['highpass'] = 1. / info['highpass']
+                    info['highpass'] = 1. / (2 * np.pi * info['highpass'])
```

Comment:
can you add as comments the urls and explanation from the issue?
so in two year when we try to understand why there is a 2pi we know why.

also can you update a test so we don't break this in the future? thx